src/pages/Calculator.tsx

```diff
@@ -146,6 +153,13 @@ export const Calculator = (): React.ReactElement => {
     calculatorData.interaction,
   )
 
+  const editScenario =
+    calculatorData.scenarioName === undefined ||
+    calculatorData.scenarioName === ''
+  const editInteractionType =
```

Comment:
nit: could this be interactionSelected or interactionEditable?

Reply:
done